book/src/drivers.md

```diff
@@ -66,6 +66,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [ionir ](https://github.com/ionir-cloud) | `ionir` | v1.2 | A Container Storage Interface (CSI) Driver for [ionir](https://www.ionir.com/) Kubernetes-Native Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Cloning
 [JuiceFS](https://github.com/juicedata/juicefs-csi-driver) | `csi.juicefs.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for JuiceFS File System | Persistent | Read/Write Multiple Pod | Yes | 
 [kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | 
+[KaiXiangTech MegaBric](https://github.com/kaixiangtech/csi-driver-flexblock) | `flexblock.csi.kaixiangtech.com` | 21.01.01.1, 21.03.04.1 | A Container Storage Interface (CSI) plugin for KaiXiangTech MegaBric Storage | Persistent | Read/Write MultiplePods | Yes | Raw Block, Expansion, Cloning
```

Comment:
Please add a README to the github repo describing how to use the driver and what it supports.

Also column 3 is for CSI spec versions supported by your CSI driver.

Add a space betwee "Multiple" and "Pods".

Reply:
update Version v1.6.0 .

Reply:
update README file into github repo and tag v1.6.0 version.

Reply:
commit add  a space betwee "Multiple" and "Pods" and version v1..6.0